pkg/consts/consts.go

```diff
@@ -187,6 +187,16 @@ const (
 	BackoffJitterDefault = 1.0
 )
 
+// LB variables for dual-stack
+var (
+	// There's no "service.beta.kubernetes.io/azure-load-balancer" annotation before these 2 new ones are added.
+	// Service.Spec.LoadBalancerIP was used and it will be deprecated.
+	ServiceAnnotationLoadBalancerIPDualStack = map[bool]string{
```

Comment:
not necessarily to be a map, just two annotations like others should be ok

Reply:
Please take a look at the example:
https://github.com/lzhecheng/cloud-provider-azure/commit/a4217028b93cc095edf25b69cc570606b30dca4d#diff-ec0e9d2cbe0713dd82a1be0894ef682a36222d21f7fb47d299b843ebbed8ba6dR77-R85

I think a map makes dualstack code cleaner without a lot of if statements.

Reply:
ok, makes sense